domains/Auth/SignIn/index.js

```diff
@@ -138,19 +138,58 @@ const SignIn = ({ navigation }) => {
   };
 
   const deleteCreds = () => {
-    deleteData('credentials');
+    deleteData('currentUser');
   };
 
-  const storeUserInformation = async (userData, userCreds) => {
+  const storeUserInformation = async (currentUser, userCreds) => {
     // store username and password
     if (userCreds) {
-      const credentials = userCreds;
-      credentials.store = 'No';
-      storeData(credentials, 'credentials');
+      const usr = userCreds;
+      usr.store = 'No';
+      storeData(usr, 'currentUser');
```

Comment:
@hopetambala I think that this store with the usr.store = 'No' will be overwritten by the populateCache, so the store portion used for the pincode will not persist

From populateCache
`    const currentUserAsync = await getData('currentUser');
      const currentOrgAsync = await getData('organization');
      // store information after sign up/sign in
      if (user) {
        if (user !== currentUserAsync) {
          await storeData(user, 'currentUser');
        }`

Reply:
@jamccomb92 check the new logic out